CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Changelog
 Unreleased
 ----------
 * Removed React dependency
+* Add options argument to themeEachCustomProperty method
```

Comment:
Changelog needs updated to add both the addition of a global terra selector as well as the enhancement to  themeEachCustomProperty to allow for a selector